spec.html

```diff
@@ -21676,7 +21684,11 @@ <h2>Syntax</h2>
         MemberExpression[?Yield, ?Await] Arguments[?Yield, ?Await]
     </emu-grammar>
     <h2>Supplemental Syntax</h2>
-    <p>When processing an instance of the production <emu-grammar>AsyncArrowFunction : CoverCallExpressionAndAsyncArrowHead `=>` AsyncConciseBody</emu-grammar> the interpretation of |CoverCallExpressionAndAsyncArrowHead| is refined using the following grammar:</p>
+    <p>When processing an instance of the production
+      <br>
+      <emu-grammar>AsyncArrowFunction : CoverCallExpressionAndAsyncArrowHead `=>` AsyncConciseBody</emu-grammar>
+      <br>
+      the interpretation of |CoverCallExpressionAndAsyncArrowHead| is refined using the following grammar:</p>
```

Comment:
As long as we're unifying formatting, the blank line below this paragraph could be removed as well.